mesonbuild/interpreter/interpreterobjects.py

```diff
@@ -381,6 +383,16 @@ def keys_method(self, args: T.List[TYPE_var], kwargs: TYPE_kwargs) -> T.List[str
     def keys(self) -> T.List[str]:
         return list(self.held_object.values.keys())
 
+    @FeatureNew('configuration_data.append_footer()', '0.63.0')
+    @noKwargs
+    def append_footer_method(self, args, kwargs):
+        if len(args) != 1:
+            raise InterpreterException("Configuration append_footer() requires 1 argument.")
+        if not isinstance(args[0], str):
+            raise InterpreterException('Configuration append_footer() argument must be a string.')
```

Comment:
These days we can use `@typed_pos_args('configuration_data.append_footer', str)` and it will check all this for you.

Reply:
Fixed locally.